db/base/collection_keys.go

```diff
@@ -17,18 +17,18 @@ import (
 	"github.com/sourcenetwork/defradb/core"
 )
 
-// MakeIndexPrefix generates a key prefix for the given collection/index descriptions
-func MakeCollectionKey(col client.CollectionDescription) core.DataStoreKey {
+// MakeDSKeyWithCollectionID generates a key prefix for the given collection/index descriptions
```

Comment:
todo: This documentation is still very broken, I'm sorry as it was broken before, but could you please fix it whilst you are here? It isnt worth a ticket IMO.

Same goes for the function below too (`MakeDocKey`)

Reply:
Haha no need to apologize, I agree it's broken and should be improved, will fix in this PR. What specifically do you want me to change that is broken? I am guessing the name `MakeDSKeyWithCollectionID` to `MakeDataStoreKeyWithCollectionID` consistent with your other suggestion?

- [ ] rename: `MakeDSKeyWithCollectionID` to `MakeDataStoreKeyWithCollectionID`

Reply:
`generates` seems like a bad word to use here, as is `key prefix`. `index descriptions` seems quite misleading too.

Reply:
Sorry I think I edited my comment very close to your last comment, and again now. Let me know if the suggested comment is better. Or if you have another suggestion.

Reply:
Ah yes, happy with your suggestion :)  Thanks Shahzad

Reply:
- [ ] Do suggestion